### PR TITLE
Feat: Updates the charge information tab

### PR DESCRIPTION
**Pull request by swaff**
WATER-2743

Updates the charge version tab to be the charge information tab

All links to incomplete pages are currently commented out.

### CODE REVIEW

File: src/shared/view/nunjucks/filters/charge-version-badge.js [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+
 const titleCase = require('title-case');
 
+const styles = {
+  current: 'completed',
+  draft: 'void',
+  approved: 'success',
+  replaced: 'inactive',
+  invalid: 'error',
+  review: 'warning'
```
Comment (Stephan-Hough) [inline src/shared/view/nunjucks/filters/charge-version-badge.js:11]: Do we need to accommodate the existing charge version status of superseded?

Comment (Stephan-Hough) [inline src/shared/view/nunjucks/filters/charge-version-badge.js:11]: Do we need to handle the existing charge version status of superseded? Should this be handled in the water service?

**Review by Stephan-Hough — CHANGES_REQUESTED** [review]

File: src/internal/views/nunjucks/view-licences/licence.njk [hunk 2]
```diff
@@ -40,7 +38,7 @@
 
           {% if showChargeVersions %}
             <li class="govuk-tabs__list-item">
-              <a class="govuk-tabs__tab" href="#charge">Charge</a>
+              <a class="govuk-tabs__tab" href="#charge">Charge information</a>
```
[the remark this answers is not in the document]
Comment (timtamimi) [reply src/internal/views/nunjucks/view-licences/licence.njk:41]: Resolved in a1c46bbf

File: src/shared/view/nunjucks/filters/charge-version-badge.js (hunk 1 above)
Comment (timtamimi) [reply src/shared/view/nunjucks/filters/charge-version-badge.js:11]: Resolved in d84c146c

Comment (timtamimi) [reply src/shared/view/nunjucks/filters/charge-version-badge.js:11]: Resolved in d84c146c

**Review by timtamimi — APPROVED** [review]

**Pull request merged**